packages/seeder/src/seeder.js

```diff
@@ -64,6 +64,9 @@ class Seeder extends EventEmitter {
     for (const key of keys) {
       // get or create hyperdrive
       const keyString = key.toString('hex')
+
+      console.log('SEEDING', keyString)
```

Comment:
this is good, next step we can focus in start pushing out all this info as better logs

Reply:
Yes, totally. Just for debug seeding flow.